query-node/mappings/content/video.ts

```diff
@@ -154,6 +162,14 @@ export async function content_VideoUpdated(ctx: EventContext & StoreContext): Pr
   // save video
   await store.save<Video>(video)
 
+  // check if video is fully active
+  const isFullyActive = isVideoFullyActive(video)
+
+  // update video counters for channel, channel category, and video category if needed
+  if (wasFullyActive !== isFullyActive) {
+    await updateVideoActiveCounters(store, video, isFullyActive)
```

Comment:
Same as in https://github.com/Joystream/joystream/pull/2954 - there needs to be special handling in case the video category has been changed / unset.
Similarly for channel category in `content_ChannelUpdated`

Reply:
The feature has been reworked. Channel and categories updates are handled now.